source/shared/core_sqlsrv.h

```diff
@@ -2184,6 +2196,17 @@ namespace core {
         }
     }
 
+    inline void SQLSetDescField(_Inout_ sqlsrv_stmt* stmt, _In_ SQLSMALLINT rec_num, _In_ SQLSMALLINT fld_id, _In_reads_bytes_opt_(str_len) SQLPOINTER value_ptr, _In_ SQLINTEGER str_len  TSRMLS_DC)
+    {
+        SQLRETURN r;
+        SQLHDESC hIpd = NULL;
+        core::SQLGetStmtAttr(stmt, SQL_ATTR_IMP_PARAM_DESC, &hIpd, 0, 0);
+        r = ::SQLSetDescField(hIpd, rec_num, fld_id, value_ptr, str_len);
+
+        CHECK_SQL_ERROR_OR_WARNING(r, stmt) {
+            throw CoreException();
+        }
+    }
 
```

Comment:
It would be nice to add a simple comment what this function is for. Also, please adhere to the coding standard re the spacing